README.md

```diff
@@ -1,6 +1,12 @@
 # Chisel3
-Chisel3 is a new Firrtl based chisel.
-It is currently in BETA VERSION, so some Chisel features may change in the coming months.
+Chisel is a new hardware construction language to support adavanced hardware design and circuit generation.
+The latest version of Chisel is Chisel3, which uses Firrtl as an intermediate hardware representation language.
+
+Chisel3 is currently in BETA VERSION, so some Chisel features may change in the coming months.
+<!---
+Is it good to have this beta warning here and no lonk to the stable Chisel 2 version?
+This warning might scare away people.
```

Comment:
I think this would/should go away soon. We're down to the final pushes for the Chisel 3 release, and when that happens this warning can go away.